src/test/java/com/google/sps/servlets/FindLocationServletTest.java

```diff
@@ -0,0 +1,48 @@
+package com.google.sps.servlets;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import com.google.sps.servlets.DBUtilities;
+import org.mockito.Mock;
+
+@RunWith(JUnit4.class)
+public final class FindLocationServletTest {
+
+  @Test
+  public void testCorrectStatusSent() {
+    HttpServletRequest request = mock(HttpServletRequest.class);       
+    HttpServletResponse response = mock(HttpServletResponse.class);    
+
+    when(request.getParameter("correlator")).thenReturn("Person1");
+
+    new FindLocationServlet().doGet(request, response);
+
+    assertEquals(HttpServletResponse.SC_OK, response.getStatus());
+  }
+
+  @Test
+  public void checkIfBadRequestStatusSentNull() {
```

Comment:
above you started the name with test and here you start with check, choose one and stay consistent

Reply:
good catch!